### ISSUE TITLE
Interface `Automattic\WooCommerce\Blueprint\Exporters\StepExporter` not found

### ISSUE THREAD

**Issue by Screenfeed** [php-stubs/woocommerce-stubs#26]
Since [v9.3.1](https://github.com/php-stubs/woocommerce-stubs/releases/tag/v9.3.1) this is the error we get:
```
Error thrown in /xxx/vendor/php-stubs/woocommerce-stubs/woocommerce-stubs.php on line 72433
while loading bootstrap file /+++/vendor/php-stubs/woocommerce-stubs/woocommerce-stubs.php:
Interface "Automattic\WooCommerce\Blueprint\Exporters\StepExporter" not found
```

**Comment by szepeviktor** [comment]
Thank you for reporting. I try to find a way in the chaos of WooCommerce.

**Comment by Screenfeed** [comment]
Thank you, be the Force be with you.

**Comment by szepeviktor** [comment]
adding `->notPath('Admin/Features/Blueprint')` which excludes JSON import/export from stubs

**Comment by szepeviktor** [comment]
there are only bad fixes: overwritten commit 3705959 and overwritten tag :(

**Comment by szepeviktor** [comment]
Thank you for reporting!
